helm/values.yaml

```
@@ -5,69 +5,138 @@ app:
     auth:
-      rootUser: tccms
+      forceNewKeys: true
+      rootUser: capn
       rootPassword:
     persistence:
-      size: "1Gi"
-    defaultBuckets: static-data,media-data
+      enabled: true
+      size: 1Gi
+    defaultBuckets: capn-static,capn-media
     provisioning:
+      enabled: true
+      resourcesPreset: "nano"
+      cleanupAfterFinished:
+        enabled: true
       extraCommands:
-        - "mc anonymous set download provisioning/static-data"
+        - "mc anonymous set download provisioning/capn-static"
 
   api:
+    enabled: true
+    replicaCount: 1
     containerPort: 80
-    command: ["gunicorn", "config.wsgi:application", "--timeout=40", "--bind", "0.0.0.0:80"]
+    command: ["gunicorn", "main.wsgi:application", "--timeout=40", "--bind", "0.0.0.0:80"]
     resources:
       requests:
-        cpu: "0.2"
+        cpu: "1"
+        memory: 0.5Gi
+      limits:
+        cpu: "4"
+        memory: 1Gi
 
-  worker:
+  argoHook:
     enabled: true
+    resources:
+      requests:
+        cpu: "0.1"
+        memory: 1Gi
+      limits:
+        cpu: "1"
+        memory: 2Gi
+    hooks:
+      db-migrate:
+        enabled: true
+        hook: PostSync
+        preserveHistory: true
+        command: ["/bin/bash", "-c"]
+        args:
+          - "./manage.py wait_for_resources --db && ./manage.py migrate"
+      collect-static:
+        enabled: true
+        hook: PostSync
+        command: ["/bin/bash", "-c"]
+        args:
+          - "./manage.py wait_for_resources --minio && ./manage.py collectstatic --noinput"
+
+  worker:
+    enabled: false
 
-  # NOTE: Check main/settings.py for all available options
   env:
+    APP_RELEASE: "{{ $.Values.image.tag }}"
+    APP_ENVIRONMENT: "{{ $.Values.environment }}"
+    TIME_ZONE: "Asia/Kathmandu"
+    DJANGO_DEBUG: "false"
     # AWS
-    AWS_S3_ENABLED: "true"
+    AWS_S3_ENABLED: "false"
     AWS_S3_ENDPOINT_URL: "https://{{ $.Values.minio.apiIngress.hostname }}/"
-    AWS_S3_STATIC_BUCKET_NAME: "static-data"
-    AWS_S3_MEDIA_BUCKET_NAME: "media-data"
+    AWS_S3_MEDIA_BUCKET_NAME: "capn-media"
+    AWS_S3_STATIC_BUCKET_NAME: "capn-static"
     # App Domain
-    SESSION_COOKIE_DOMAIN: "{{ required \".Values.env.SESSION_COOKIE_DOMAIN\" nil }}"
-    CSRF_COOKIE_DOMAIN: "{{ required \".Values.env.CSRF_COOKIE_DOMAIN\" nil }}"
+    APP_TYPE: "WEB"
+    APP_DOMAIN: "{{ required \".app.env.APP_DOMAIN\" nil }}"
+    FRONTEND_DOMAIN: "{{ required \".app.env.FRONTEND_DOMAIN\" nil }}"
+    SESSION_COOKIE_DOMAIN: "{{ required \".app.env.SESSION_COOKIE_DOMAIN\" nil }}"
+    CSRF_COOKIE_DOMAIN: "{{ required \".app.env.CSRF_COOKIE_DOMAIN\" nil }}"
+    # Sentry
+    SENTRY_ENABLED: "false"
+    SENTRY_TRACES_SAMPLE_RATE: "0.2"
+    SENTRY_PROFILE_SAMPLE_RATE: "0.2"
 
-  # NOTE: Check main/settings.py for all available options
   secrets:
-    SECRET_KEY: "{{ required \".Values.secrets.SECRET_KEY\" nil }}"
+    DJANGO_SECRET_KEY: "thoh-2i##o2j^6--_sg-oq!j9ohqipx3_a-c2+"
```

```suggestion
  SECRET_KEY: "{{ required \".Values.secrets.SECRET_KEY\" nil }}"
```
```suggestion
    DJANGO_SECRET_KEY: "{{ required \".Values.secrets.DJANGO_SECRET_KEY\" nil }}"
```
Let's not add this on values.yaml instead throw validation error